tensorboard/plugins/pr_curve/pr_curve_plugin_test.py

```diff
@@ -0,0 +1,94 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Tests the op that generates pr_curve summaries."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import json
+import os
+
+import numpy as np
+import tensorflow as tf
+
+from tensorboard.backend.event_processing import event_multiplexer
+from tensorboard.plugins.pr_curve import pr_curve_op
+from tensorboard.plugins.pr_curve import pr_curve_plugin_data_pb2
+
+
+class PrCurveTest(tf.test.TestCase):
+
+  def setUp(self):
+    self.logdir = self.get_temp_dir()
+    tf.reset_default_graph()
+
+  def test1Class(self):
+    # Generate summaries for showing PR curves in TensorBoard.
+    with tf.Session() as sess:
+      pr_curve_op.op(
+          tag='tag_bar',
+          labels=tf.constant([True, False, True, False], dtype=tf.bool),
+          predictions=tf.constant([0.8, 0.6, 0.4, 0.2], dtype=tf.float32),
+          num_thresholds=10)
+      merged_summary_op = tf.summary.merge_all()
+      foo_directory = os.path.join(self.logdir, 'foo')
+      writer = tf.summary.FileWriter(foo_directory)
+      writer.add_summary(sess.run(merged_summary_op), 1)
+      writer.close()
+
+    # Create a multiplexer for reading the data we just wrote.
+    multiplexer = event_multiplexer.EventMultiplexer()
+    multiplexer.AddRunsFromDirectory(self.logdir)
+    multiplexer.Reload()
+
+    # Verify that the metadata was correctly written.
+    accumulator = multiplexer.GetAccumulator('foo')
+    tag_content_dict = accumulator.PluginTagToContent('pr_curve')
+    self.assertListEqual(['tag_bar'], tag_content_dict.keys())
+    serialized_data = tag_content_dict['tag_bar']
+    plugin_data = pr_curve_plugin_data_pb2.PrCurvePluginData()
+    plugin_data.MergeFromString(serialized_data)
```

Comment:
no need to declare variable serialized_data,
plugin_data.MergeFromString(tag_content_dict['tag_bar'])